### ISSUE TITLE
Peer to peer code review

### ISSUE THREAD

**Issue by patriciachrysy** [patriciachrysy/using-bootstrap#1]
Leave some comments for me guys and also some tips for the responsiveness of the smallest screen size

**Comment by nyatindopatrick** [comment]
Good work! The larger screen looks good to me, though the smaller screens still lacks responsiveness. To fix responsiveness of the columns I would suggest this:
- for a columns use this example:
`class="col-lg-4 col-md-6 col-sm-12"`
this changes the size of the grid according to the screen size.

This header button is also misbehaving on smaller screens
![Screen Shot 2020-02-12 at 10 15 35](https://user-images.githubusercontent.com/48393059/74311988-c043a500-4d81-11ea-81b1-29b3722e474a.png)

**Comment by Kyle-Law** [comment]
Great job designing the navbar to have that right-edges effect! Here are my few remarks:

At largest screen size:
- Spaces between the navbar and main news could be removed by giving them spacing bootstrap classes. Check out: https://getbootstrap.com/docs/4.4/utilities/spacing/

At Medium Screen size:
- 'Sign up for newsletter' should be centralized. (Try mx-auto, or text-center maybe?)

Tips for the smallest screen size:
- The sequence is **TOP STORY > MORE STORIES > CULTURE & TRAVEL**, these could be done with flexbox `order` property.
- 'More Stories' and 'Latest News' contained 2 columns, where pic on the left and description on the right.
- Take note that currently at screens below 576px, the layout switches back to desktop size. 

You're almost there, keep it up! 👍

**Comment by dili021** [comment]
-Pick a screen size and start going through the whole page, comparing and sizing elements 1 by 1. I suggest going to medium size as that's the first transition, even though this is backwards when working with bootstrap.
-There is a solid horizontal scroll on the medium size
-can't take a screenshot for some reason but you have some redundant bootstrap classes, example column two, the top story section. You have `class="col-5 col-lg-5 col-xl-5 col-md-7 col-sm-12 mx-auto"` col-5 makes no sense as col-n with no size means from xs and above until defined differently. 5/12 columns is very small for xs screens. 
col-xl-5 is also not necessary as col-lg-5 is saying stay at 5 columns width from lg and above (which is only xl in this case. 
So if you say for example col-sm-5 col-lg-4 the column will be defined as follows:
from xs it will resize itself as a regular flex item, dividing the width with other columns in the row.
If you define any breakpoint it will default to col-12 (full width) to the sizes bellow 
Then next screen size is sm, it changes to 5 columns as defined. Next size is md - nothing defined so it keeps 5 from previous. Next is lg so it becomes 4 and stays there for the rest of it's bootstrapped expansion life :D 

So basically you just define the breakpoints you want. If you want an element to be full screen on phones and tablets, but have three of them in a row on computers, you just say col-lg-4 on the three elemens and that's it! 

Sorry if it sounds like I'm over-explaining it, but understanding it this way helped me a lot.
